packages/bootstrap-4/src/SelectWidget/SelectWidget.tsx

```diff
@@ -52,8 +55,7 @@ const SelectWidget = ({
         value={typeof value === "undefined" ? emptyValue : value}
         required={required}
         multiple={multiple}
-        disabled={disabled}
-        readOnly={readonly}
+        disabled={disabled && readonly}
```

Comment:
Why were these consolidated?

Reply:
If they have the same effect, then I think it shouldn't it be `||`?

Reply:
Yeah, I think they are actually different? Not 100% sure about how this widget works on bootstrap

Reply:
They were consolidated to fix and issue because readOnly is not supported on the bootstrap select widget... and yeah it should be an ||